perma_web/perma/templates/user_management/create-link.html

```diff
@@ -150,6 +150,8 @@ <h3 id="link-list-header" class="body-ah">Your Perma Links</h3>
 
 {% if request.user.is_staff %}
   {% flag "vue-dashboard" %}
+    <div class="vanilla-div">Vanilla JavaScript Count: None</div>
```

Comment:
Note: The value you see rendered on the page should be 0, not none. I wrote this to verify that I could run the `watch` function that updates this count would run eagerly — when the vue app first mounts, as well as for any future updates to the Vue store.